app/screens/login/login.js

```diff
@@ -383,8 +394,6 @@ export default class Login extends PureComponent {
                         <ErrorText error={this.state.error}/>
                         <TextInput
                             ref={this.loginRef}
-                            value={this.props.loginId}
-                            onChangeText={this.props.actions.handleLoginIdChanged}
```

Comment:
Why would we update state and re-render a component when is not needed?

Reply:
And the benefit? I think you are overthinking this one

Reply:
It has not changed since TextInput was added to RN, this also prevents going through the bridge for every keystroke.

I’m very confident about this, and in any case if it changes is going to be easily identifiable